src/mongocxx/include/mongocxx/v_noabi/mongocxx/read_preference.hpp

```diff
@@ -205,8 +226,10 @@ class read_preference {
     ///   A reference to the object on which this member function is being called.  This facilitates
     ///   method chaining.
     ///
-    MONGOCXX_ABI_EXPORT_CDECL(read_preference&)
-    tags(bsoncxx::v_noabi::array::view_or_value tag_set_list);
+    read_preference& tags(bsoncxx::v_noabi::array::view_or_value tag_set_list) {
+        _rp.tags(bsoncxx::v_noabi::to_v1(tag_set_list.view()));
+        return *this;
+    }
 
     ///
     /// Sets or updates the tag set list for this read_preference.
```

Comment:
Not introduced in this PR, but suggested drive-by fix of incorrect comment:

```suggestion
    /// Gets the tag set list for this read_preference.
```